terraform/gcp/modules/monitoring/fulcio/fulcio_alerts.tf

```diff
@@ -174,3 +174,75 @@ resource "google_monitoring_alert_policy" "ca_service_cert_quota" {
   notification_channels = local.notification_channels
   project               = var.project_id
 }
+
+### K8s Alerts
+
+resource "google_monitoring_alert_policy" "fulcio_k8s_pod_restart_failing_container" {
+  # In the absence of data, incident will auto-close in 7 days
+
+  alert_strategy {
+    auto_close = "604800s"
+  }
+
+  combiner = "OR"
+
+  conditions {
+    condition_threshold {
+      filter     = "metric.name=\"k8s_pod/restarting-failed-container\" resource.type=\"k8s_pod\""
+      duration   = "600s"
```

Comment:
For the first pass, these alerts will fire if the error message is found in logs for at least ten minutes. Let me know if we would like to change this threshold.